src/main/java/teammates/logic/core/SendgridService.java

```diff
@@ -37,9 +36,10 @@ public Sendgrid parseToEmail(EmailWrapper wrapper) throws JSONException {
      * {@inheritDoc}
      */
     @Override
-    public void sendEmail(EmailWrapper wrapper) throws JSONException, IOException {
-        Sendgrid email = parseToEmail(wrapper);
-        email.send();
+    public void sendEmail(EmailWrapper wrapper) throws SendGridException {
+        Email email = parseToEmail(wrapper);
+        SendGrid sendgrid = new SendGrid(Config.SENDGRID_APIKEY);
+        sendgrid.send(email);
```

Comment:
I think in the [example shown](https://cloud.google.com/appengine/docs/flexible/java/sending-emails-with-sendgrid#example), they do an additional check for response code 200, do we need to include that check here and throw an exception if it's not 200?